src/index.js

```diff
@@ -1,41 +1,57 @@
-import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
-import addItem from './modules/addItem.js'
+/* eslint-disable import/no-cycle */
+import options from './assets/options.png';
```

Comment:
- [ ] Hello, Don't disable the linter, because the linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs